javascript/core/flow-control/loops-part-2.md

```diff
@@ -128,8 +128,8 @@ var myArray = [1,2,"3","item"];
 To exit a loop, you can use the ??? keyword.
 
 
-* break
-* stop
-* finish
-* end
+* `break`
+* `stop`
+* `finish`
+* `end`
```

Comment:
the answers already display as if enclosed by backticks,so we don't need to enclose in backticks anymore

![image](https://user-images.githubusercontent.com/19802557/77747332-6687f900-7016-11ea-9b77-8b17946a7980.png)


```suggestion
* break
* stop
* finish
* end
```